modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,143 @@
+= Multi-factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.
+
+Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. Instead, SSO users should configure multi-factor authentication through their identity provider.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using at least one of the following authentication methods:
+
+* One-time password generator
```

Comment:
We are supposed to use the terminology "Third-party TOTP authenticator apps"

Reply:
```suggestion
* Third-party TOTP authenticator apps
```